### PR TITLE
Feature/payments container

### PR DESCRIPTION
**Pull request by Njoe00**
adding competent for payment types for website

### CODE REVIEW

File: src/components/PaymentsContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import react from "react";
+import Image from "next/image";
+
+export default function PayMentsContainer() {
```
Comment (MKCMMSK) [inline src/components/PaymentsContainer.tsx]: Naming

File: src/components/PaymentsContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  return (
+    <div className="bg-background-payments-colour flex items-center flex-col w-full">
+      <div className="text-black flex justify-center text-2xl font-sans">
+        Safe, easy payment options
+      </div>
+      <ul className="flex justify-center flex-row items-center">
+        {PaymentIcons.map((icon, index) => (
+          <Image className="flex items-center m-10" src={icon.icon} height={icon.size[1]} width={icon.size[0]} alt="test" key={index} />
+        ))}
+      </ul>
```
Comment (MKCMMSK) [inline src/components/PaymentsContainer.tsx]: Why is ul being used with li? Seems odd an unordered list isn't using list elements.

Another thing to note is that the icon.size[0] and icon.size[1] seems like it's been poorly designed. Why isn't it potentially icon.height, icon.width? Or if for some reason you wanted it that nested, icon.size.height and icon.size.width.

**Review by MKCMMSK — CHANGES_REQUESTED** [review]
Some odd choices

File: src/components/PaymentsContainer.tsx (hunk 1 above)
Comment (Njoe00) [reply src/components/PaymentsContainer.tsx]: addressed.

File: src/components/PaymentsContainer.tsx (hunk 2 above)
Comment (Njoe00) [reply src/components/PaymentsContainer.tsx]: changed li to div and adding width and height to data object.

**Review by MKCMMSK — APPROVED** [review]

**Pull request merged**